tests/test_iotedgedev_solution.py

```diff
@@ -1,26 +1,21 @@
 import json
 import os
 import platform
-import pytest
 import shutil
 import time
+from unittest import mock
```

Comment:
>unittest [](http://example.com/codeflow?start=5&length=8)

seems unused?

Reply:
That's being used on line 437
`@mock.patch.dict(os.environ, {"CONTAINER_REGISTRY_SERVER": "localhost:5000"})`